code/modules/clothing/head/hardhat.dm

```diff
@@ -21,6 +21,9 @@
 	heat_protection = SLOT_HEAD
 	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
 	max_pressure_protection = FIRESUIT_MAX_PRESSURE
+	material = /decl/material/solid/metal/steel
+	matter = list(/decl/material/solid/metal/plasteel = MATTER_AMOUNT_TRACE)
```

Comment:
plasteel seems a bit too costly for something so wide-spread, and it doesn't even have that good armor, steel-tier values.
heck I'd even go for plastic with steel reinforcment.

Reply:
Ill fuck with this later, Truthfully I don't have the energy right now given I have classes starting in like 4 days